subprojects/test-runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/GetSdkVersion.kt

```diff
@@ -0,0 +1,37 @@
+package com.avito.runner.service.worker.device.adb
+
+import com.avito.android.Result
+import com.avito.runner.service.worker.device.Serial
+import com.avito.runner.service.worker.device.adb.listener.AdbDeviceGetSdkListener
+import com.avito.utils.ProcessRunner
+import java.time.Duration
+
+class GetSdkVersion(
```

Comment:
Fix number two
Extracted logic for `get sdk prop`

Reply:
Is it because `AdbDevice` became too big, or there is also another reason?

Reply:
Because we need to react outside for example delete this